### PR TITLE
Add code of args task for merge into dev/args whit Christian/args

### PR DESCRIPTION
**Pull request by clopezbaya**

### CODE REVIEW

**Review by J0c4 — CHANGES_REQUESTED** [review]
Looks good, just few comments about english. Also, it would be good to have behavior implemented with schema as well (did I miss it?)

**Comment by J0c4** [comment]
Hello @clopezbaya , any updates on the changes requested? (use english instead of spanish in code). We need those changes in order to approve an merge this PR

**Comment by clopezbaya** [comment]
Okay i am sorry , I forget now update please

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

File: katas/args/christian/parserClass.test.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+const aux = new Parser('hola mundo');
+test('Class Init recived String correctly', () => {
+    const result = aux.args;
+    expect(result).toBe('hola mundo');
+});
+
+const aux1 = new Parser('hola mundo');
+test(' Parser Work Completly', () => {
+    const result = aux1.init();
+    expect(result).toBe('Ningun Argumento Coincide');
```
[the remark this answers is not in the document]
Comment (clopezbaya) [reply katas/args/christian/parserClass.test.js]: All the problems are already solved

File: katas/args/christian/main.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { Parser } from './parserClass.js';
+
+process.stdout.write('ingresa los argumentos: ');
```
[the remark this answers is not in the document]
Comment (clopezbaya) [reply katas/args/christian/main.js]: All the problems are already solved

**Comment by J0c4** [comment]
hi @clopezbaya , just noticed, I detect some copy paste here, who is the real author?

**Comment by clopezbaya** [comment]
Yes I am the owner of the code, I made it from scratch

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Review by J0c4 — APPROVED** [review]

**Pull request merged**